### DISCUSSION TITLE
Can't upgrade from v20.2.0 to v20.3.0

### DISCUSSION THREAD

**Discussion by dev-venkatesh** [OpenMage/magento-lts#3738] [Questions & Answers]
I am not sure on how to upgrade from v20.2.0 to v20.3.0.

I executed the below command to upgrade to the latest version in my OpenMage (Magento) root directory.
`composer update`
But the above command not upgraded to the latest version v20.3.0.

Please suggest the commands how to upgrade from v20.2.0 to v20.3.0 via composer.

**Comment by sreichel** [comment]
Can you pls share your composer.json?

**Reply by dev-venkatesh** [reply]
Attached

**Comment by pquerner** [comment]
`composer require openmage/magento-lts:20.3.0`

**Comment by dev-venkatesh** [comment]
I am getting the below error after running the command in my root directory.
Command:
`composer require openmage/magento-lts:20.3.0`

Error:
`Root package 'openmage/magento-lts' cannot require itself in its composer.json`

**Comment by dev-venkatesh** [comment]
My **composer.json** file.
[composer.json](https://github.com/OpenMage/magento-lts/files/14006315/composer.json)

**Comment by dev-venkatesh** [comment]
Initially I installed manually not via composer. Is it a problem?

**Reply by pquerner** [reply]
When you used git, no. Then you can just checkout the latest tag you want and work from there.

`git checkout tags/<version>` (for instance 20.3.0)

Although its still a little unclear (to me) what you did.

**Reply by fballiano** [reply]
if you installed manually you can't run composer require openmage/magento-lts:20.3.0, you need a totally different composer.json.

in this case just upgrade manually.

otherwise rebuild the composer.json like this one https://github.com/fballiano/mcommerce/blob/main/composer.json
